articles/hdinsight/kafka/kafka-troubleshoot-full-disk

```diff
@@ -0,0 +1,83 @@
+
+```
+# Command to sort the directories based on size:
+du -hs * | sort -rh | head -5 
+```
+ 
+3. Backup the files which are older than the new retention time.
+
+```
+# Command to find files older than 4 days:
+find . -maxdepth 1 -mtime +4 -type f -exec ls -l "{}"  \; 
+```
+
+> [!WARNING] 
+> Make sure **NOT** to remove/move `_meta.properties_` file in "/data_disk_*/kafka-logs" directory. Deleting `_meta.properties_` file causes broker ids to get updated when Kafka restarts. Topics which were created with old broker ids become under replicated or offline so the cluster falls in a bad state.
+
+```
+# Command to move files older than 4 days:
+find . -maxdepth 1 -mtime +4 -type f -exec mv "{}" /destination/directory/ \;
+```
+4. When there is some free space available, you can restart the brokers with new retention time. This will clean the older logs and free up some space on disk.
+ 
+> [!IMPORTANT]
+> Sometimes taking a backup might not be an option because either OS disk might get full or just overload other kafka disks, in that case we might have to delete the files older than the retention time.
+
+## Resolution
+
+Even though you can decrease the retention time, it will not be scalable if you want to add more topics in the cluster or if the load or amount of data ingested increases.
+To avoid the above the scenario these are the following options:
+
+- If the partitions is too big, increase the number of partitions for heavy topics
+
+> [!NOTE] 
+> But increasing the partitions in an existing topic doesn't reorganize the data in the topic. You have to manually copy data from a old low partition topic to a new higher partition topic in the first. 
+- Scale out the cluster and re-balance all the partitions across multiple disks.
+
+- Create a cluster with bigger sku and more no. of disks attached.
+
+## Next steps
+
+If you didn't see your problem or are unable to solve your issue, visit one of the following channels for more support:
+
+* Get answers from Azure experts through [Azure Community Support](https://azure.microsoft.com/support/community/).
+
+* Connect with [@AzureSupport](https://twitter.com/azuresupport) - the official Microsoft Azure account for improving customer experience. Connecting the Azure community to the right resources: answers, support, and experts.
+
+* If you need more help, you can submit a support request from the [Azure portal](https://portal.azure.com/?#blade/Microsoft_Azure_Support/HelpAndSupportBlade/). Select **Support** from the menu bar or open the **Help + support** hub. For more detailed information, review [How to create an Azure support request](../../azure-portal/supportability/how-to-create-azure-support-request.md). Access to Subscription Management and billing support is included with your Microsoft Azure subscription, and Technical Support is provided through one of the [Azure Support Plans](https://azure.microsoft.com/support/plans/).
```

Comment:
These lines are already part of a templated include file: /hdinsight/includes/hdinsight-troubleshooting-next-steps.md

Just embed the include and remove the text in the Next steps.

```suggestion
[!INCLUDE [troubleshooting next steps](../includes/hdinsight-troubleshooting-next-steps.md)]
```